suite-native/module-accounts/src/screens/AccountDetailScreen.tsx

```diff
@@ -74,6 +74,18 @@ export const AccountDetailScreen = memo(() => {
         setAreTokensIncluded(prev => !prev);
     }, []);
 
+    const listHeaderComponent = useMemo(
```

Comment:
Wouldn't it be better to apply memo to the component itself?

Reply:
Probably yes :D